### PR TITLE
Chore/repo improves

### PR DESCRIPTION
**Pull request by vanbasten17**
## Description
Little improvements to our repo:
* Fix broken links for Botonic Logo
* Add License: MIT to every package
* Add keywords to every package (to be more searchable by `npm search` and `npm`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 <p align="center">
   <a  href="https://botonic.io/">
-    <img alt="Node.js" src="https://botonic.io/images/botonic_react_logo-p-500.png" width="150"/>
+    <img alt="Node.js" src="https://botonic.io/img/botonic-logo.png"f width="150"/>
```
Comment (MarcosCA) [inline README.md:11]: .png"f
that "f" ?

File: packages/botonic-plugin-watson/package.json [hunk 2]
```diff
@@ -12,5 +13,12 @@
...
     "ibm-watson": "^5.1.0"
   },
-  "devDependencies": {}
+  "devDependencies": {},
+  "keywords": [
+    "javascript",
+    "chatbots",
+    "conversational apps",
+    "watson",
+    "segment"
```
Comment (MarcosCA) [inline packages/botonic-plugin-watson/package.json:22]: is correct segment here?

**Review by MarcosCA — CHANGES_REQUESTED** [review]

File: packages/botonic-plugin-watson/package.json (hunk 2 above)
Comment (vanbasten17) [reply packages/botonic-plugin-watson/package.json]: good catches, fixed 👍

**Review by MarcosCA — APPROVED** [review]

**Review by asastre — APPROVED** [review]
Nice update!
Would it be better to use words in singular rather than plural as keywords? I've search a little bit but I haven't found any clear information about it.

**Pull request merged**